scripts/superdesk-authoring/authoring.js

```diff
@@ -303,6 +305,23 @@
             }
         };
 
+        this.rewrite = function (item, authoringWorkspace) {
```

Comment:
I need it in authoring service (rewrite), and since authoringWorkspace uses authoring service, I cannot add it in authoring, so I did it this way..